src/utils/form-ui.ts

```diff
@@ -116,7 +133,7 @@ function changeHostProtocol(property: string) {
  * @param document
  * @param partsPath
  */
-async function turtleToFormUi(document: any) {
+async function turtleToFormUi(document: any, language: string) {
```

Comment:
Does this mean you're enforcing the library consumer to set a language here? In other words: they'll have to always ask the user for their language? (My browser is set to English, but I often write in Dutch.) And what if there are form fields that aren't really localisable? For example, what if I'm setting my age?

Also note that the data type of a field with a language set is different from one that does not have a language set (`http://www.w3.org/1999/02/22-rdf-syntax-ns#langString` vs `http://www.w3.org/2001/XMLSchema#string`), so if e.g. solid-ui does not set the language, enforcing a language might make it incompatible with solid-ui.